pyproject.toml

```
@@ -23,3 +34,20 @@ before-all = [
 ]
 # Skip musllinux builds for the moment
 skip = "*-musllinux_*"
+
+[tool.scikit-build]
```

```suggestion
[tool.scikit-build]
minimum-version = "build-system.requires"
```

And I'd probably make that 0.10 or 0.11.